drafts/2019-05-16-restful-config-viewing-and-validation-service.adoc

```diff
@@ -0,0 +1,327 @@
+---
+layout: post
+title: "Testing database connections in Open Liberty apps with REST APIs"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/30153613
+author_github: https://github.com/nmittles
+seo-title: Testing database connections in Open Liberty apps with REST APIs - OpenLiberty.io
+seo-description: Looking for an easy and fast way to test your database connections? You can now validate your connections by invoking REST endpoints provided by the configValidator 1.0 beta feature. In addition to being easy and fast, these REST endpoints will exercise the same code paths as your applications, giving you confidence in your server configuration.
+blog_description: "Looking for an easy and fast way to test your database connections? You can now validate your connections by invoking REST endpoints provided by the configValidator 1.0 beta feature. In addition to being easy and fast, these REST endpoints will exercise the same code paths as your applications, giving you confidence in your server configuration."
+---
+= Testing database connections in Open Liberty apps with REST APIs
+Nathan Mittlestat <https://github.com/nmittles>
+
+== Testing database connections in Open Liberty apps with REST APIs
+
+Looking for an easy and fast way to test your database connections? You can now validate your connections by invoking REST endpoints provided by the configValidator 1.0 beta feature. In addition to being easy and fast, these REST endpoints will exercise the same code paths as your applications, giving you confidence in your server configuration. 
```

Comment:
instead of "easy and fast" maybe just say "simple". Sound a bit less infomercial-y

Reply:
Done